server/routes/utils/dataReportHelpers.ts

```diff
@@ -0,0 +1,212 @@
+import { async } from 'rxjs/internal/scheduler/async';
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import { DATA_REPORT_CONFIG } from './constants';
+
+const esb = require('elastic-builder');
+const moment = require('moment');
+const converter = require('json-2-csv');
```

Comment:
Add dependencies to `package.json`, use `import .. as .. from.. ` syntax that typescript can recognize

Reply:
Done